src/main/kotlin/ar/edu/unahur/obj2/semillasAlViento/Parcela.kt

```diff
@@ -2,44 +2,48 @@ package ar.edu.unahur.obj2.semillasAlViento
 
 class Parcela(val ancho: Int, val largo: Int, val horasSolPorDia: Int) {
   val plantas = mutableListOf<Planta>()
-  var cantidadPlantas = 0
 
   fun superficie() = ancho * largo
   fun cantidadMaximaPlantas() =
-    if (ancho > largo) ancho * largo / 5 else ancho * largo / 3 + largo
+    if (ancho > largo) this.superficie() / 5 else this.superficie() / 3 + largo
 
   fun plantar(planta: Planta) {
-    if (cantidadPlantas == this.cantidadMaximaPlantas()) {
-      println("Ya no hay lugar en esta parcela")
-    } else if (horasSolPorDia > planta.horasDeSolQueTolera() + 2) {
-      println("No se puede plantar esto acá, se va a quemar")
+    if (plantas.size == this.cantidadMaximaPlantas()) {
+      throw Exception("Ya no hay lugar en esta parcela")
+    } else if (horasSolPorDia >= planta.horasDeSolQueTolera() + 2) {
+      throw Exception("No se puede plantar esto acá, se va a quemar")
     } else {
       plantas.add(planta)
```

Comment:
Está bien, aunque te podés ahorrar los `else` - al arrojarse la excepción el flujo de ejecución se corta y nunca llega al código de abajo:
```suggestion
    if (plantas.size == this.cantidadMaximaPlantas()) {
      throw Exception("Ya no hay lugar en esta parcela")
    } 
    if (horasSolPorDia >= planta.horasDeSolQueTolera() + 2) {
      throw Exception("No se puede plantar esto acá, se va a quemar")
    }
    plantas.add(planta)
```